ansible/playbooks/monitoring.yml

```diff
@@ -244,6 +244,15 @@
                 summary: "Self-Hosted runner {{ '{{ $labels.runner }}' }} in repo {{ '{{ $labels.repo }}' }} is offline"
                 description: "GitHub is reporting that the self-hosted runner {{ '{{ $labels.runner }}' }} in the repo {{ '{{ $labels.repo }}' }} is offline. This could be caused by the machine hosting the runner having problems, or by the runner software being out of date."
 
+        - name: playground
+          rules:
+            - alert: PlaygroundDown
+              expr: sum(irate(playground_request_duration_seconds_count[5m])) == 0
```

Comment:
```suggestion
              expr: sum(increase(playground_request_duration_seconds_count[5m])) == 0
```

Reply:
I'll have to figure out the meaningful difference here I suppose.